types/ApiRequests.ts

```diff
@@ -1,5 +1,5 @@
-import { FeedbackStatus, FeedbackType, SocialMediaType, UserAccessType } from './Enums';
 import { Uuid } from '.';
+import { FeedbackStatus, FeedbackType, SocialMediaType, UserAccessType } from './Enums';
```

Comment:
remove feedback status from here when you remove it from the request

Reply:
There's a UpdateFeedbackStatusRequest in that file still, may need to keep it